### ISSUE TITLE
Add isDateOverlapped method or something like this

### ISSUE THREAD

**Issue by mRWSEI** [joaomatossilva/DateTimeExtensions#98]
I guess it would be usefull to add isOverlappedDate / isDateOverlapped helper method to check for dates.
Here's implementation proposal (yeah I know its easy but I have to look for stackoverflow each time to find it would be easier with method like this)

```
public bool isDateOverlapped(DateTime DateAStart, DateTime DateAEnd, DateTime DateBStart, DateTime DateBEnd)
{
return DateAStart < DateBEnd && DateBStart < DateAEnd;
}

```

**Comment by joaomatossilva** [comment]
I don't see why not :)

Care for submitting a PR?

**Comment by mRWSEI** [comment]
Wonderful, 
I can try - never done this before. Looks like easy way to do this first time ;)

Does syntax/naming is appropriate?

**Comment by joaomatossilva** [comment]
Yes. Try to see other methods and follow the same Naming/Case Convention. This can also be useful for other projects contributions

**Comment by mRWSEI** [comment]
Ok i will try do my best but need few more days I am going to implement all 4 methods and tests for them regarding to this image
![DateTime Ranges](https://i.stack.imgur.com/0c6q0.png)

**Comment by mrzechowicz** [comment]
done #99
